apps/docker/README.md

```diff
@@ -1,27 +1,41 @@
-# Run AzerothCore with Docker
+# Docker
```

Comment:
just make sure the wiki is up-to-date

Reply:
updated https://github.com/azerothcore/wiki/pull/920 with commit https://github.com/azerothcore/wiki/pull/920/commits/b3557df61f6747447bffc1d13555337a3af5c82e